views/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-qti-item",
-    "version": "0.4.0",
+    "version": "0.4.1",
```

Comment:
Sorry, my mistake. No need to version bump extension, please revert it.

But bundles should be here, yes.

Reply:
Done